data/json/recipes/recipe_others.json

```diff
@@ -1588,6 +1588,17 @@
     "qualities": [ { "id": "HAMMER", "level": 2 } ],
     "components": [ [ [ "sheet_metal", 1 ], [ "scrap", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "arm_xlsplint",
+    "copy-from": "arm_splint",
```

Comment:
It seems `copy-from` for xl splint recipe didn't work properly:

![image](https://user-images.githubusercontent.com/16213433/44583831-9ffbe100-a7af-11e8-842e-9e83183e7c6a.png)


![image](https://user-images.githubusercontent.com/16213433/44583816-91adc500-a7af-11e8-82e5-c00534958926.png)

Reply:
Fuck. I know using copy-from is good practice and all, but man, it would be so much easier to just write everything out.